dev/tests/api-functional/testsuite/Magento/GraphQl/CatalogCustomer/PriceTiersTest.php

```diff
@@ -117,6 +117,34 @@ public function testGetLowestPriceForGuest()
         $this->assertEquals(round(7.25, 2), $this->getValueForQuantity(8, $itemTiers));
     }
 
+    /**
+     * @magentoApiDataFixture Magento/Catalog/_files/three_simple_products_with_tier_price.php
+     */
+    public function testGetCorrectDisplaingTierPriceForProducts()
```

Comment:
Most probably we'll have to improve the test coverage, by having at least 1 product without tierPrices, in order to make sure it doesn't return anything.

Reply:
**DONE ** 
Added one product  without tier price.